dynamic-demo-plugin/README.md

```diff
@@ -28,6 +30,29 @@ the script, for example:
 yarn http-server -a 127.0.0.1
 ```
 
+In another terminal window, run:
+
+1. `oc login` (requires [oc](https://console.redhat.com/openshift/downloads) and an [OpenShift cluster](https://console.redhat.com/openshift/create))
+2. `yarn run start-console` (requires [Docker](https://www.docker.com) or [podman 3.2.0+](https://podman.io))
+
+This will run the OpenShift console in a container connected to the cluster
+you've logged into. The plugin HTTP server runs on port 9001 with CORS enabled.
+Navigate to <http://localhost:9000/example> to see the running plugin.
```

Comment:
Does the demo plugin have a different URL or does it also use this route?

Reply:
@spadgett It was a copy/paste error from the template repository. Has been updated